ddtrace/ext/ci.py

```diff
@@ -60,6 +62,9 @@
 # Version of the ddtrace library
 LIBRARY_VERSION = "library_version"
 
+# CI Visibility env vars used for pipeline correlation ID
+CI_ENV_VARS = "_dd.ci.env_vars"
```

Comment:
```suggestion
_CI_ENV_VARS = "_dd.ci.env_vars"
```

If this is for internal use only, we can consider making this a private variable.